anonymize.py

```diff
@@ -1,18 +1,15 @@
 from presidio_anonymizer import AnonymizerEngine
 from presidio_analyzer import AnalyzerEngine
 from presidio_anonymizer.entities import OperatorConfig
-
+from utils import pdf_to_txt
 
 #read txt file
-text=''
-with open("text.txt",'r') as file:
-    text=file.read()
+text=pdf_to_txt("pdf")
```

Comment:
I don't get it. Your file is supposed to be named `pdf`?

Can you instead create a function with the file name as input?